buffer/src/main/java/io/netty/buffer/ByteBufProcessor.java

```diff
@@ -16,10 +16,14 @@
 
 package io.netty.buffer;
 
-public interface ByteBufProcessor {
+/**
+ * Use {@link ByteVisitor} directly where interfaces permit.
+ */
+public interface ByteBufProcessor extends ByteVisitor {
```

Comment:
@trustin - I started down this path yesterday.  Wasn't sure if we wanted to have duplication of the interfaces that use `ByteBufProcessor`...but I'm cool with it.